packages/flame_audio/lib/flame_audio.dart

```diff
@@ -112,12 +134,21 @@ class FlameAudio {
     String sound, {
     required int maxPlayers,
     int minPlayers = 1,
-  }) {
+    AudioContext? audioContext,
+  }) async {
+    audioContext ??= _defaultAudioContext;
+    // TODO(gustl22): Probably set context for each player individually,
+    //  as soon as functionality is supported.
```

Comment:
IIRC that was how I wanted to impl but iOS didn't support it so I had to make it global, but that was half a decade ago so my memory fails me

Reply:
In fact, for iOS it's still global. We changed that behavior to align it along with Android, so on iOS, if setting it for one player, it is actually applied to all players. So the dev experience is much worse, needing to differentiate the cases themselves, than actually experiencing no change at all for that given method, which we got a lot of tickets for. It's all documented though.

Reply:
yeah I recall there was some sort of nuisance between platforms, as often is the case...